x-pack/legacy/plugins/lens/public/metric_visualization_plugin/metric_expression.test.tsx

```diff
@@ -9,7 +9,7 @@ import { LensMultiTable } from '../types';
 import React from 'react';
 import { shallow } from 'enzyme';
 import { MetricConfig } from './types';
-import { FieldFormat } from 'ui/registry/field_formats';
+import { FieldFormat } from '../../../../../../src/plugins/data/public';
```

Comment:
Please explain this change. 'ui/registry/field_formats' export only fieldFormats. Why you replaced it to src/plugins/data/public. Are you sure that need it at all?

Reply:
Agree. It has no sense. Removed